.github/workflows/sycl-containers.yaml

```diff
@@ -55,6 +55,10 @@ jobs:
             file: ubuntu2204_intel_drivers
             tag: latest
             build_args: "use_latest=false"
+          - name: Intel Drivers Ubuntu 24.04 Docker image
+            file: ubuntu2404_intel_drivers
+            tag: latest
+            build_args: "use_latest=false"
```

Comment:
Also, one question: the tag says "latest" but the `build_args` says "use_latest=false". What does the `build_args` used for?

Reply:
yeah this is confusing, `latest` means `latest IGC driver`, which is the unstable one, but the `latest` image version is the latest validated one (using stable igc). I didn't come up with that, just porting existing stuff to 24.04, thanks.